crates/bevy_render/src/camera/camera.rs

```diff
@@ -106,7 +128,7 @@ pub fn camera_system<T: CameraProjection + Component>(
         added_cameras.push(entity);
     }
     for (entity, mut camera, mut camera_projection) in queries.q0().iter_mut() {
-        if let Some(window) = windows.get(camera.window) {
+        if let Some(window) = camera.get_window().and_then(|window| windows.get(window)) {
```

Comment:
Here we should check if either the window has changed, or the image handle had any `AssetEvent::Modified` events so that changing the size of a render image updates the cameras projection.

Reply:
Done.